packages/slate/test/interfaces/Range/equals/not-equal.tsx

```diff
@@ -20,6 +21,7 @@ export const input = {
       path: [0, 1],
       offset: 0,
     },
+    placeholder: 0,
```

Comment:
You'll need a dedicated test for extra attributes as `anchor` and `focus` are already not-equal in this test